### PR TITLE
PERF: Remove support for librte

### PR DESCRIPTION
**Pull request by shamisp**
Why: Librte does not exits anymore and it is not maintained.

### CODE REVIEW

**Comment by gleon99** [comment]
Can remove also mentions of RTE in:
- `glossaries.rst`
- `src/tools/perf/configure.m4` ("TODO")
- `src/tools/perf/{,lib/}Makefile.am` (flags)
- `libperf.h`

**Comment by shamisp** [comment]
`libperf.h` - the rte struct is used for MPI
`glossaries.rst ` - the term is still relevant since we use it in the code.

**Review by yosefe — DISMISSED** [review]

**Review by yosefe — DISMISSED** [review]

File: src/tools/perf/Makefile.am [hunk 1]
```diff
@@ -23,9 +23,9 @@ ucx_perftest_SOURCES = \
 	perftest_run.c \
 	perftest_params.c
 
-ucx_perftest_CPPFLAGS = $(BASE_CPPFLAGS) $(RTE_CPPFLAGS)
+ucx_perftest_CPPFLAGS = $(BASE_CPPFLAGS)
```
Comment (gleon99) [inline src/tools/perf/Makefile.am:26]: Also the same under `lib/Makefile.am`?

Comment (shamisp) [reply src/tools/perf/Makefile.am:26]: Good catch. I updated and squashed to safe the time.

**Comment by shamisp** [comment]
the error looks irrelevant

**Review by yosefe — APPROVED** [review]

**Pull request merged**